akka-projection-r2dbc-integration/src/test/scala/akka/projection/r2dbc/R2dbcTimestampOffsetStoreSpec.scala

```diff
@@ -729,13 +774,6 @@ class R2dbcTimestampOffsetStoreSpec
       offsetStore
         .saveOffset(OffsetPidSeqNr(TimestampOffset(startTime.plus(evictInterval), Map(p4 -> 1L)), p4, 1L))
         .futureValue
-      offsetStore
-        .saveOffset(
-          OffsetPidSeqNr(
-            TimestampOffset(startTime.plus(evictInterval).plus(JDuration.ofSeconds(1)), Map(p4 -> 1L)),
-            p4,
-            1L))
-        .futureValue
```

Comment:
I'm not sure what the original intention with this one was, but it was never saved because it was a duplicate. Same p4 and seqNr 1 as above. I changed `isDuplicate` to also look at timestamp when same seqNr and then this test failed. By removing this the test is kind of as it was before.